test/e2e/activationscale/activation_scale_test.go

```diff
@@ -0,0 +1,90 @@
+//go:build e2e
+// +build e2e
+
+/*
+Copyright 2022 The Knative Authors
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package e2e
+
+import (
+	"fmt"
+	"strconv"
+	"testing"
+	"time"
+
+	"github.com/davecgh/go-spew/spew"
+	vegeta "github.com/tsenart/vegeta/v12/lib"
+	pkgTest "knative.dev/pkg/test"
+	"knative.dev/serving/pkg/apis/autoscaling"
+	rtesting "knative.dev/serving/pkg/testing/v1"
+	"knative.dev/serving/test"
+	"knative.dev/serving/test/e2e"
+)
+
+const activationScale = 5
+
+func TestActivationScale(t *testing.T) {
+	t.Parallel()
+
+	ctx := e2e.SetupSvc(t, autoscaling.KPA, autoscaling.Concurrency, 6, 0.7,
+		rtesting.WithConfigAnnotations(map[string]string{
+			autoscaling.ActivationScaleKey:        strconv.Itoa(activationScale),
+			autoscaling.InitialScaleAnnotationKey: "0", // avoid having to wait for scale to zero
```

Comment:
I think we should drop the use of initial scale in this test - otherwise it's sorta testing two things